modules/deploy/pages/deployment-option/cloud/manage-billing/aws-commit.adoc

```diff
@@ -0,0 +1,43 @@
+= Use with AWS Commitment
+:description: Subscribe to Redpanda Cloud in AWS Marketplace with discounted commitment billing.
+
+With a usage-based billing commitment, you sign up for a minimum spend amount and get a discount on your committed usage. This allows you to subscribe to Redpanda Cloud through AWS Marketplace to quickly provision Redpanda Dedicated clusters. 
+
+Commitments are minimums. If you use more than your committed amount, you can continue using Redpanda Cloud without interruption. You will be charged at your discounted rate for usage beyond the committed amount until the end of your commitment term. 
+
+[IMPORTANT]
+====
+When you sign up for Redpanda Cloud through AWS Marketplace, you can only create clusters on AWS. 
```

Comment:
I'm reading this as, "You can only use AWS Marketplace to create AWS Clusters." A user could still use GCP to set up a GCP cluster, right? Or will Redpanda licensing not support that?

Mainly just looking for some clarification on this!

Reply:
Users will only be able to create clusters in AWS through AWS Marketplace. Similary they will be able to create clusters in GCP through GCP Marketplace only.  They won't be able to create an AWS cluster through GCP Marketplace for example.